### ISSUE TITLE
Use a database such as PostgreSQL to manage data files

### ISSUE THREAD

**Issue by chadalen** [chadalen/astraeus#35]
***PostgreSQL*** I used heavily throughout the industry and it's open-source. It's way faster than ***MongoDB*** and offers non-relational storage support such as JSON just like MongoDB.

**Comment by IanWONeill** [comment]
NoSQL

**Comment by chadalen** [comment]
@IanWONeill Yeah I'm still playing around with different databases. I have a few in mind still such as CouchDB, MongoDB and PostgreSQL

**Comment by IanWONeill** [comment]
Yeah CouchDB is what ive been looking into. Though the only problem facing is our userbase, they see 'SQL' and have a heart murmur. SQL is what everyone should be using its factual... But yet they use CSV. Let me know what you decide to do because im going to copy you haha. Also... Ive yet too come accross a sqldb using json. How would you format the json? Just remove all whitespace and stuff? Make it "ugly"-json.

But keep it SQL!!

**Comment by chadalen** [comment]
@IanWONeill Well that's what the wonders of a NoSQL database are such as MongoDB. Instead of using SQL it uses JSON documents. My project already consists of JSON documents for all of my data. Which is ideal for this project. I'm not a fan of SQL for this kind of project because like you mentioned most of the userbase won't know how to use it, plus SQL is typically found in relational-databases and for the sake of this project relational-database would not be a good way to go. MongoDB is extremely easy to use and most of the community uses JSON or should so it's easier to adapt to.

I like CouchDB, but it's so new. However I like how it uses Javascript as it's query language which is nice. Javascript is a powerful language and it's syntax is easy to learn.
